frontend/src/metabase/lib/entities.js

```diff
@@ -36,7 +36,7 @@
  *
  * Say we want to create a "books" entity, to be able to fetch a list of "books".
  *
- * Add the following line to `frontend/src/metabase/entities.index.js`:
+ * Add the following line to `frontend/src/metabase/entities.index.ts`:
```

Comment:
Why only change this line but not also line 43? But I'm not really sure this is deliberate.

Reply:
It wasn't intended, fixed